### PR TITLE
app-emulation/fuse: bump to version 1.5.6

### PR DESCRIPTION
**Pull request by atomsymbol**
Please merge. Thanks.

### CODE REVIEW

File: app-emulation/fuse/fuse-1.5.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (mgorny) [inline app-emulation/fuse/fuse-1.5.6.ebuild]: Homepage says `GPL-2+`.

**Comment by atomsymbol** [comment]
fuse-1.5.6.tar.gz/COPYING is GPL-2.

**Comment by mgorny** [comment]
COPYING is just a copy of license as required by GPL terms. The actual license is defined in copyright notices on top of files, which frequently state that 'or (...) newer version'.

**Comment by atomsymbol** [comment]
Just a note: I already updated the ebuild license to GPL-2+.

**Comment by mgorny** [comment]
Thanks, I'll merge it now.

**Pull request closed without merging**